quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/HttpPropertyMappers.java

```diff
@@ -109,6 +114,10 @@ public static PropertyMapper[] getHttpPropertyMappers() {
         };
     }
 
+    private static BiFunction<Optional<String>, ConfigSourceInterceptorContext, Optional<String>> validatePath(String key) {
+        return (value, context) -> value.filter(v -> v.equals(context.proceed(key).getValue())).map(p -> p.replace("\\", "/"));
```

Comment:
It's valid, but unlikely, for unix files and directories to contain the \ character.  This probably needs to be done only for windows.  Or generally require uses to be consistent - and error out with mixed file separators.

Reply:
I can think of one edge case on Unix platform and that's when a space in a directory/file name is escaped.

So I proceeded with the backslash replacement just for the windows platform and with no transformation for Unix. I also added a separate test case for each platform.

Reply:
I also retested it on Windows, just to be on a safe side.